web/store/index.js

```diff
@@ -97,7 +97,10 @@ export default new Vuex.Store({
       const { locations } = state.locationsSelection;
       return locations[0];
     },
-    locationActive(state) {
+    locationActive(state, getters) {
+      if (state.locationMode === LocationMode.SINGLE) {
```

Comment:
tempted to say we should get rid of the `location` getter, then, and just inline it here!  probably a task for after request management stuff; I've submitted it as #590 .